### ISSUE TITLE
Emmet override the key binding for atom sniplets

### ISSUE THREAD

**Issue by anandshah05** [emmetio/emmet-atom#430]
When I enable emmet, I can not do sniplets with tab.
When I disable emeet, tab start working for sniplets

Atom - 1.7.3

**Comment by aaronmw** [comment]
Yeah, for some reason using "tab" to accept auto-complete suggestions is broken for me too, but only in CSS/SCSS files...

**Comment by heyarne** [comment]
I have the same problem. This is with emmet 2.4.3 and atom 1.9.9. If there is any additional info I can provide I'm happy to do so.
